src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -267,51 +276,19 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
     return (this.getComponentValidate && !this.getSpecialResolutionFormValid)
   }
 
-  /**
-   * The minimum date that can be entered.
-   * Can't be earlier than incorporation date.
-   * Date only - time is 12:00 am Pacific.
-   */
-  get resolutionDateMin (): Date {
-    /** TODO: Needs to be after the most recent filing date, I don't think the business founding date is correct
-     * Will be fixed in 13231 */
-
-    // convert date to YYYY-MM-DD and back to date
-    // to set time to 12:00 am Pacific time
-    const date = this.apiToDate(this.getBusinessFoundingDate)
-    const yyyyMmDd = this.dateToYyyyMmDd(date)
-    return this.yyyyMmDdToDate(yyyyMmDd)
-  }
-
-  /**
-   * The maximum date that can be entered (today).
-   * Date only - time is 12:00 am Pacific.
-   */
-  get resolutionDateMax (): Date {
-    return this.yyyyMmDdToDate(this.getCurrentDate)
-  }
-
-  /** Validations rules for resolution date field. */
-  get resolutionDateRules (): Array<Function> {
-    return [
-      (v: string) => !!v || 'Resolution date is required',
-      (v: string) =>
-        this.isValidDateRange(this.resolutionDateMin,
-          this.resolutionDateMax,
-          v) ||
-        `Date should be between ${this.dateToPacificDate(this.resolutionDateMin, true)} and
-         ${this.dateToPacificDate(this.resolutionDateMax, true)}`
-    ]
+  /** The minimum resolution date that can be entered (incorporation date). */
+  get resolutionDateMin (): string {
+    /**
+     * TODO: Needs to be after the most recent filing date, I don't think the business founding date is correct
+     * Will be fixed in 13231
+     */
+    const date = this.apiToDate(this.getBusinessFoundingDateTime)
+    return this.dateToYyyyMmDd(date)
   }
```

Comment:
I moved this up into a readonly property.

I also removed the "valid date range" check since the min/max already limit the values that the date picker allows the user to select.

Ditto for signatureDateRules below.